gsl_programmation/models.py

```diff
@@ -173,6 +203,8 @@ class ProgrammationProjet(models.Model):
         verbose_name="Date de dernière modification", auto_now=True
     )
 
+    objects = ProgrammationProjetManager()
```

Comment:
Tu ne peux pas directement faire comme ça ?

```suggestion
    objects = models.Manager.from_queryset(ProgrammationProjetQuerySet)
```

Reply:
J'ai vu qu'après, je viens de tester et ça ne fonctionne pas !